src/main/java/edu/hm/hafner/analysis/parser/OwaspDependencyCheckParser.java

```diff
@@ -0,0 +1,98 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+
+/**
+ * OWASP dependency check JSON report parser.
+ */
+public class OwaspDependencyCheckParser extends JsonIssueParser {
+    private static final long serialVersionUID = -1369431674771459756L;
+
+    private static final String SUPPORTED_SCHEMA_VERSION = "1.1";
+
+    private static final String SCHEMA_VERSION = "reportSchema";
+    private static final String DEPENDENCIES = "dependencies";
+    private static final String VULNERABILITIES = "vulnerabilities";
+    private static final String SEVERITY = "severity";
+    private static final String NAME = "name";
+    private static final String CVSSV2 = "cvssv2";
+    private static final String CVSSV3 = "cvssv3";
+    private static final String ACCESS_VECTOR = "accessVector";
+    private static final String ATTACK_VECTOR = "attackVector";
+    private static final String DESCRIPTION = "description";
+
+    private final Map<String, Severity> severityMap = new HashMap<>();
+
+    /**
+     * Create instance.
+     */
+    public OwaspDependencyCheckParser() {
+        super();
+        severityMap.put("LOW", Severity.WARNING_LOW);
+        severityMap.put("MEDIUM", Severity.WARNING_NORMAL);
+        severityMap.put("HIGH", Severity.WARNING_HIGH);
+        severityMap.put("CRITICAL", Severity.ERROR);
+    }
+
+    @Override
+    protected void parseJsonObject(final Report report, final JSONObject jsonReport, final IssueBuilder issueBuilder) {
+        final String schemaVersion = jsonReport.getString(SCHEMA_VERSION);
+        if (!SUPPORTED_SCHEMA_VERSION.equals(schemaVersion)) {
+            throw new ParsingException("unsupported report schema version: %s", schemaVersion);
+        }
+        if (jsonReport.has(DEPENDENCIES)) {
```

Comment:
Does this indicate a wrong configuration? Then maybe a warning message would make sense.

Reply:
Yes, this indicates misconfiguration. I'm going to remove this check. The parse error should be expressive enough.